### PR TITLE
[docs] Update camera plugin usage

### PR DESCRIPTION
**Pull request by alanjhughes**
# Why
Some properties were missing in the plugin example usage.

# How
Updates the example.

# Test Plan
Ran the docs locally

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/camera.mdx [hunk 1]
```diff
@@ -64,6 +66,13 @@ You can configure `expo-camera` using its built-in [config plugin](/config-plugi
       description: 'A string to set the [`NSMicrophoneUsageDescription`](#ios) permission message.',
       default: '"Allow $(PRODUCT_NAME) to access your microphone"',
     },
+    {
+      name: 'recordAudioAndroid',
+      platform: 'android',
+      description:
+        'A boolean that determines whethed to enable the `RECORD_AUDIO` permission on Android',
```
Comment (fobos531) [inline docs/pages/versions/unversioned/sdk/camera.mdx:73]:
```suggestion
        'A boolean that determines whether to enable the `RECORD_AUDIO` permission on Android',
```

File: docs/pages/versions/v50.0.0/sdk/camera-next.mdx [hunk 2]
```diff
@@ -67,6 +69,13 @@ You can configure `expo-camera` using its built-in [config plugin](/config-plugi
       description: 'A string to set the [`NSMicrophoneUsageDescription`](#ios) permission message.',
       default: '"Allow $(PRODUCT_NAME) to access your microphone"',
     },
+    {
+      name: 'recordAudioAndroid',
+      platform: 'android',
+      description:
+        'A boolean that determines whethed to enable the `RECORD_AUDIO` permission on Android',
```
Comment (fobos531) [inline docs/pages/versions/v50.0.0/sdk/camera-next.mdx:76]:
```suggestion
        'A boolean that determines whether to enable the `RECORD_AUDIO` permission on Android',
```

File: docs/pages/versions/v50.0.0/sdk/camera.mdx [hunk 3]
```diff
@@ -64,6 +66,13 @@ You can configure `expo-camera` using its built-in [config plugin](/config-plugi
       description: 'A string to set the [`NSMicrophoneUsageDescription`](#ios) permission message.',
       default: '"Allow $(PRODUCT_NAME) to access your microphone"',
     },
+    {
+      name: 'recordAudioAndroid',
+      platform: 'android',
+      description:
+        'A boolean that determines whethed to enable the `RECORD_AUDIO` permission on Android',
```
Comment (fobos531) [inline docs/pages/versions/v50.0.0/sdk/camera.mdx:73]:
```suggestion
        'A boolean that determines whether to enable the `RECORD_AUDIO` permission on Android',
```

File: docs/pages/versions/unversioned/sdk/camera-next.mdx [hunk 4]
```diff
@@ -67,6 +69,13 @@ You can configure `expo-camera` using its built-in [config plugin](/config-plugi
       description: 'A string to set the [`NSMicrophoneUsageDescription`](#ios) permission message.',
       default: '"Allow $(PRODUCT_NAME) to access your microphone"',
     },
+    {
+      name: 'recordAudioAndroid',
+      platform: 'android',
+      description:
+        'A boolean that determines whethed to enable the `RECORD_AUDIO` permission on Android',
```
[the remark this answers is not in the document]
Comment (amandeepmittal) [reply docs/pages/versions/unversioned/sdk/camera-next.mdx:76]: There's also a missing period at the end of `description`. Updated the above suggestion.

**Review by amandeepmittal — APPROVED** [review]

**Review by Simek — APPROVED** [review]
👌

**Pull request merged**